ui-tests/src/test/java/plugins/WarningsNextGenerationPluginTest.java

```diff
@@ -0,0 +1,621 @@
+package plugins;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A high number of imports can indicate a high degree of coupling within an object.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)